client/src/components/Preview/Preview.js

```diff
@@ -98,7 +114,12 @@ class Preview extends Component {
     }
 
     // Show iframe preview
-    return <iframe className="flexbox-area-grow preview__iframe" src={previewUrl} />;
+    return (<iframe
+      style={{ visibility: this.state.frameLoaded ? 'visible' : 'hidden' }}
+      className="flexbox-area-grow preview__iframe"
+      src={previewUrl}
+      onLoad={() => this.setState({ frameLoaded: true })}
```

Comment:
Do you want to make a method like `setFrameLoaded(loaded = true)` and use that here and line 29?

Reply:
Done